### PR TITLE
Added workflows for obagent and ob-configserver

### PR DESCRIPTION
**Pull request by powerfooI**
## Summary

1. Added workflows for obagent and ob-configserver
2. Removed redundant double quote for obproxy startup param values

### CODE REVIEW

File: .github/workflows/release-obconfigserver.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: release obconfigserver
```
Comment (whhe) [inline .github/workflows/release-obconfigserver.yml:1]: Should use `ob-configserver`.

Comment (powerfooI) [reply .github/workflows/release-obconfigserver.yml]: OK

File: .github/workflows/release-obagent.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - name: Build and push obagent w/o cache
+        uses: docker/build-push-action@v2
+        with:
+          context: ./obagent
+          platforms: linux/amd64,linux/arm64
+          file: ./obagent/Dockerfile
+          push: true
+          tags: ${{ vars.DOCKER_PUSH_BASE }}/obagent:${{ steps.set_version_vars.outputs.version }}
+          build-args: |
+            VERSION=${{ steps.set_version_vars.outputs.version }}
```
[the remark this answers is not in the document]
Comment (powerfooI) [reply .github/workflows/release-obagent.yml]: OK

**Review by whhe — APPROVED** [review]
LGTM

**Pull request merged**